uktrade/data-hub-frontend
# Estimated land date reminders

## Description of change

Adds a new page to list reminders for approaching estimated land dates. Note that this url is not linked to from anywhere so we can continue to test it.

Moves the reminders settings page to a new url from /reminders to /reminders/settings

Note that the pages for outstanding propositions on no recent interactions have not been created yet so those links are 404s.

## Test instructions

- Create some reminders for yourself in django admin
- Go to /reminders/estimated-land-date - this should include a list of reminders if you have any

## Screenshots

![image](https://user-images.githubusercontent.com/1234577/172641811-43530e68-63df-4db4-9c05-d76e7a1c545a.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
